.github/actions/update-charm-pins/main.py

```diff
@@ -9,14 +9,14 @@
 from httpx import Client
 from ruamel.yaml import YAML
 
-yaml = YAML(typ="rt")
+yaml = YAML(typ='rt')
```

Comment:
I think the changes to this file are probably good (more in line with our Ruff format). However, I don't think they belong in this PR. Can you please revert them here?

Reply:
Opened [a PR to fix this](https://github.com/canonical/operator/pull/1278), which I've hit too - it happens if you use the `tox -e fmt` we provide :)

Reply:
#1278 so that github links the two.

Reply:
#1278 has been merged; an alternative to reverting this file would be to rebase.